### ISSUE TITLE
The rule template for YAML documents should contain some keysRULE TEMPLATE does not work with the given example.

### ISSUE THREAD

**Issue by rahul096d** [sbaudoin/sonar-yaml#53]
The above rule template was extended for creating a new rule. The following values were provided : 
![image](https://user-images.githubusercontent.com/55573263/111126031-412e2b00-8598-11eb-95c9-b6a46557101d.png)
After running the sonar scan for the below yaml file no code sonar issue was raised. The contents of the yaml are same as mentioned in the given example in the rule template as mentioned below : 
```yaml
apiVersion: v1
kind: Pod
metadata:
    labels:
        test: liveness
    name: liveness-http
spec:
    containers:
        - name: liveness
            image: k8s.gcr.io/liveness
            args:
                - /server
            initialDelaySeconds: 3
            periodSeconds: 3
```

**Comment by sbaudoin** [comment]
Hello,

There's a typo in the sample YAML code, so the rule should raise a syntax error ("syntax error: mapping values are not allowed here") instead of a violation of the rule. The correct code is as follows:

```yaml
apiVersion: v1
kind: Pod
metadata:
    labels:
        test: liveness
    name: liveness-http
spec:
    containers:
        - name: liveness
          image: k8s.gcr.io/liveness
          args:
              - /server
          initialDelaySeconds: 3
          periodSeconds: 3
```

I fix the doc and I add a unit test to demonstrate this.
